pcrs/experiment/install-vm.sh

```diff
@@ -0,0 +1,32 @@
+set -ex
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/vm-config
+BUTANE_CONFIG="${BASE_DIR}/example.bu"
+IGNITION_CONFIG="${BASE_DIR}/example.ign"
+IGNITION_DEVICE_ARG=(--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${IGNITION_CONFIG}")
+mkdir -p ${BASE_DIR}
+pushd ${BASE_DIR}
+curl -L -O https://builds.coreos.fedoraproject.org/prod/streams/next/builds/42.20250623.1.0/x86_64/fedora-coreos-42.20250623.1.0-qemu.x86_64.qcow2.xz
+xz -d fedora-coreos-42.20250623.1.0-qemu.x86_64.qcow2.xz
+mv fedora-coreos-42* ${IMAGE}
+
+cat << EOF > ${BUTANE_CONFIG}
+variant: fcos
+version: 1.6.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        -
+EOF
+butane --pretty --strict ${BUTANE_CONFIG} > $IGNITION_CONFIG
+
+# Setup the correct SELinux label to allow access to the config
+chcon --verbose --type svirt_home_t ${IGNITION_CONFIG}
+
+virt-install --connect="qemu:///session" --name="${VM_NAME}" --vcpus="${VCPUS}" --memory="${RAM_MB}" \
+        --os-variant="fedora-coreos-$STREAM" --import --graphics=vnc \
+        --disk="size=${DISK_GB},backing_store=${IMAGE}" \
+        --tpm=emulator \
+        --boot uefi,loader=/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd,loader.readonly=yes,loader.secure=yes,nvram_template=/usr/share/edk2/ovmf/OVMF_VARS.secboot.fd \
+        --network bridge=virbr0 "${IGNITION_DEVICE_ARG[@]}"
```

Comment:
This failed for me -- no bridge=virbr0 for qemu:///session. I replaced it with --network user ( + adding password_hash to .bu file to be able to login).